### PR TITLE
Fix tags in metadata.yaml files: declare cdk tags

### PR DESCRIPTION
**Pull request by alafanechere**
## What
Relates to https://github.com/airbytehq/airbyte-internal-issues/issues/2849

## How
We want:
*  a single `language` tag to be defined
* a `cdk` tag to assess if the connector is using the low-code, file cdk or python cdk.

## User impact
This change might have an impact on our internal dashboards.
@alex-gron @AnilEMaharjan could you tell me how impactful it is?

### CODE REVIEW

**Comment by alafanechere** [comment]
* **#35670** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/35670?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alafanechere and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by AnilEMaharjan** [comment]
@alafanechere -- we are going to need to rewrite one of our macros to handle this change since we were specifically looking for `language:low-code` for one of our columns and for a few other changes, but it seems like we should be able to handle this. 

@alex-gron - not sure if I am missing anything else here

**Comment by alex-gron** [comment]
All good from my end! Thanks for tagging us, @alafanechere 🙏

File: airbyte-integrations/connectors/destination-duckdb/metadata.yaml [hunk 1]
```diff
@@ -19,11 +19,16 @@ data:
   releases:
     breakingChanges:
       0.3.0:
-        message: "This version uses the DuckDB 0.9.1 database driver, which is not backwards compatible with prior versions. MotherDuck users can upgrade their database by visiting https://app.motherduck.com/ and accepting the upgrade. For more information, see the connector migration guide."
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/destination-duckdb/metadata.yaml:22]: Is this purely a formatting change? Are you absolutely certain those are safe and won't blow up in publishing?

**Review by natikgadzhi — APPROVED** [review]
@alafanechere were those changes automated, or manual? I see formatting checks that I'm not absolutely sure about — it would be safer to isolate just the tags changes. 

The original issue had a list of possible options, but I only see `language` and `cdk` tags, that's it. Are we removing all other tags in favor of these two?

**Comment by natikgadzhi** [comment]
@alex-gron @AnilEMaharjan @alafanechere One concern: if we merge this, what will happen to all of the connectors views and questions on Metabase?

Will the columns on Metabase be renamed, or will the underlying processing change to keep the columns the same? Would you like to align BQ columns with what we have in metadata?

**Comment by AnilEMaharjan** [comment]
> @alex-gron @AnilEMaharjan @alafanechere One concern: if we merge this, what will happen to all of the connectors views and questions on Metabase?
> 
> Will the columns on Metabase be renamed, or will the underlying processing change to keep the columns the same? Would you like to align BQ columns with what we have in metadata?

@natikgadzhi - since we got a heads up on this, we handled the area where things were going to break (i.e. where `language:low-code` changed to `- cdk:low-code`) However, we still need to model this properly since we were previously just using a like statement searching for the value `language:low-code` as opposed to actually taking the value. 

TLDR: nothing will break, but I do need to make some updates.

**Review by katmarkham — APPROVED** [review]

**Comment by octavia-approvington** [comment]
Lets merge it <br/> sheep thumbs up

**Review by octavia-approvington — APPROVED** [review]
